views/components/BarGraphTooltip/index.jsx

```diff
@@ -14,10 +14,10 @@ const BarGraphTooltip = ({
 }) => {
   if (!active || !payload?.length) return null
 
-  const siteName = useSiteName ? label : SITE_NAMES[label]
+  const siteName = useSiteName ? label : SITE_NAMES[label] || label
   const countsTotal = useSiteName
     ? studyTotals[label].count
-    : studyTotals[SITE_NAMES[label]].count
+    : studyTotals[SITE_NAMES[label] || label].count
```

Comment:
can we use the `siteName` constant here?

```suggestion
    : studyTotals[siteName].count
```

Reply:
Actually I think we can clean this up even further by changing

```js
  const countsTotal = useSiteName
    ? studyTotals[label].count
    : studyTotals[SITE_NAMES[label] || label].count
```

to

```js
  const countsTotal = studyTotals[siteName].count
```

Reply:
The snippet looks good, and I want to get in your head, where is siteName coming from..?

Reply:
Ah I see it at the top, thanks

Reply:
[9dbeb38](https://github.com/AMP-SCZ/dpdash/pull/736/commits/9dbeb387aee249e8301096f872943cfa463512d0)